### ISSUE TITLE
IDE Local - Changes made revert to the previous state in the Action Blocks and Design too

### ISSUE THREAD

**Issue by richenri** [FlutterFlow/flutterflow-issues#4032]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

https://github.com/user-attachments/assets/677341eb-1d3a-49fe-ac73-bb03210dc898

My project is large with about 150 screens and for a few versions now, changes made mainly to Action Blocks revert to a previous state. This also happens when designing screens or when I change a component property. It seems to me that due to the size of the project, while synchronization is being done the behavior happens. It is impossible to work this way because we have to do the same implementation several times.

### Expected Behavior

Keep changes

### Steps to Reproduce

See de video

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT40lsmBw5dgrs9J1oSJbsFFimU5Jm16R+cVjup7aCosfZT2BLMcec/RVERoUs++YXNXMWeGp2oW0a2LioftF8ArFyueR75IwqlxEDqXRny5MrWaFJWgOnR8BZ9jJ2q+35+33RBANfhtdGQC3DmMHNiydDLvQee2OjA8Da/LZO4=

### Visual documentation

https://github.com/user-attachments/assets/66f6032c-d38d-44b0-a797-9debff31fe99

### Environment

```markdown
- FlutterFlow version: 5.05+
- Platform: Desktop
- Browser name and version: N/A
- Operating system and version affected: Windows
```

**Comment by TemplarDesign** [comment]
I can confirm, experiencing the same behavior here. After making several changes, the changes would revert.

https://www.loom.com/share/8994f918ade442f686d55dacd3c19e23

**Comment by j-mcq** [comment]
Our team are having the same issues. It's incredibly fustrating to keep losing work.

**Comment by TemplarDesign** [comment]
Any update on this? It's impossible to work on apps. I was unable to work on any client projects yesterday and today is looking like the same... 
We all have deadlines to meet and cannot work on this current version of FlutterFlow.

**Comment by TemplarDesign** [comment]
Many other users are experiencing this same issue:
https://community.flutterflow.io/ask-the-community/post/unreliability-issues-version-5-0-5---loading-icon-is-not-a-good-update-sxQYeTGhOXmywLo

**Comment by TemplarDesign** [comment]
Another: https://community.flutterflow.io/ask-the-community/post/flutterflow-windows-editor-is-the-new-web-editor---poor-performance-5KpxxDUWjtD1kwX

**Comment by TemplarDesign** [comment]
@agreaves

**Comment by TemplarDesign** [comment]
Duplicate issue: https://github.com/FlutterFlow/flutterflow-issues/issues/4035
So even more people are experiencing this same issue.

**Comment by deepanchal** [comment]
Any updates on this issue? It's very frustrating losing changes and having to make the same change twice. It's been happening since yesterday for me.

**Comment by Alezanello** [comment]
Hello everyone!

The engineering team is already aware of this issue. Thank you all for your reports and for your patience as we work on resolving it!

**Comment by TemplarDesign** [comment]
When will this fix be deployed? @br0nk

**Comment by cecmarco** [comment]
On my end the issue still persist. On MacOS Desktop app (Version 5.0.6) and in Chrome/Safari/Firefox. Tried them all just to be sure.

**Comment by TemplarDesign** [comment]
Same here @cecmarco 
The issue does still happen for me, but way less frequently.

**Comment by JeffersonFarFan** [comment]
hello is there any update for Mac or chrome ?

**Comment by impctfy** [comment]
Hallo this issue still exist. it frustrated alot on the works. should be fixed as fast as possible. flutter flow == Fast & Reliable . this issue makes no sense to that meaning . please fix the issue

**Comment by cecmarco** [comment]
> Hello everyone!
> 
> The engineering team is already aware of this issue. Thank you all for your reports and for your patience as we work on resolving it!

The issue is closed here but still persists. Any updates?

**Comment by deepanchal** [comment]
The issue is still happening less frequently now

**Comment by impctfy** [comment]
What do you mean less frequently? It happens everytime changes made.

**Comment by richenri** [comment]
@Alezanello any update about the correction? The latest version available for Windows is still 5.0.5 from October 8th. It is impossible to work with. We have to implement the same thing several times and the application always closes. When do you intend to release the new version? Thanks
